Dockerfile

```diff
@@ -1,3 +1,9 @@
+FROM golang:1.22.0 AS build
+WORKDIR /build
+COPY ./ ./
+RUN go mod verify
+RUN CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' .
```

Comment:
Can you please extract this command into a tiny shell script file and use it both here and in the GH action?
I'm a little worried these might drift apart.

Reply:
Added a make target for the binary build